sensorlib/src/CarlaCDASimAdapter.py

```diff
@@ -72,7 +72,9 @@ def __get_simulated_sensor(self, infrastructure_id, sensor_id):
 
     def __get_detected_objects(self, infrastructure_id, sensor_id):
         detected_objects = self.__api.get_detected_objects(infrastructure_id, sensor_id)
-        return SimulatedSensorUtils.serialize_to_json(detected_objects)
+        return_json = str(SimulatedSensorUtils.serialize_to_json(detected_objects))
+        print(">>>>>>>>>printing " + return_json)
```

Comment:
Why are we formatting print statements like this. Is this just to find them easier? If so we may want to use logger and log formatting to include line and file number.

https://stackoverflow.com/questions/533048/how-to-log-source-file-name-and-line-number-in-python

Reply:
removed all the print logs